### PR TITLE
HT-2909: Remove Kubernetes stuff

### PR DESCRIPTION
**Pull request by aelkiss**
Moved to https://github.com/hathitrust/ht_tanka; see https://github.com/hathitrust/ht_tanka/pull/9

### CODE REVIEW

**Comment by aelkiss** [comment]
See especially https://github.com/hathitrust/holdings-backend/blob/HT-2909-k8s-cleanup/routine_tasks.md

I'm interested in feedback on whether we should move this file to Confluence -- I am leaning towards doing that as it references things beyond this repository including a lot of internal paths, people & configuration

File: README.md [hunk 1]
```diff
@@ -117,22 +97,15 @@ copies of the same item).
...
-To load a scrubbed file:
+To load a scrubbed file in development:
   * Get the file(s) you want from `/htapps/mwarin.babel/phdb_scripts/data/loadfiles/`
   * Add UUIDs for tracking whether the individual line has been processed: `bin ruby/add_uuid.rb infile > outfile`
   * `docker-compose run --rm dev bundle exec bin/add_print_holdings.rb outfile`
 
-
-
 ## K8s Cronjob
 `kubectl create -f cron_job.yaml`
```
Comment (jsteverman) [inline README.md:106]: OT: I don't think this ever made it from the old concordance repo to holdings-backend. The cronjob in ht_tanka remains commented out. https://github.com/hathitrust/ht_tanka/pull/9/files#diff-d37db7a43061041cef4bba8b70897d4bdaa3b8675e491a31c89a56a395295de6

Comment (aelkiss) [reply README.md:106]: Thanks -- I was wondering where that ended up. I was thinking we were waiting to re-enabled it until we fully automate the whole process -- do we want it enabled now?

**Comment by jsteverman** [comment]
> See especially changes to this file: https://github.com/hathitrust/holdings-backend/blob/HT-2909-k8s-cleanup/routine_tasks.md
> 
> I'm interested in feedback on whether we should move this file to Confluence -- I am leaning towards doing that as it references things beyond this repository including a lot of internal paths, people & configuration

I agree, it should be moved to Confluence.

**Comment by aelkiss** [comment]
https://github.com/hathitrust/holdings-backend/blob/main/routine_tasks.md moved to https://hathitrust.atlassian.net/wiki/spaces/HAT/pages/2032107525/Routine+Tasks+for+Holdings+System

**Pull request merged**